client/app/routes/collection/add.js

```diff
@@ -19,35 +19,40 @@ export default Ember.Route.extend({
                     allowOpen: true,
                     open: true,
                     showValidationIndicator: true,
-                    value: undefined
+                    value: undefined,
+                    isSectionValid: false
```

Comment:
I meant to rename this to `isSectionSaved`, since the save icon is tracking save state and not validation